packages/transaction-controller/src/TransactionControllerIntegration.test.ts

```diff
@@ -2351,6 +2351,54 @@ describe('TransactionController Integration', () => {
     });
   });
 
+  describe('when changing rpcUrl of networkClient', () => {
```

Comment:
per our convo in slack
```suggestion
  describe('when adding a new networkClient', () => {
```

Reply:
i think we can leave it as `changing` since its doing add/remove

Reply:
I don't think it's immediately obvious that change actually means add and remove since it's not possible to change the rpcUrl of an existing networkClient. 

We actually may not need this test since really what we are testing is a remove and an add which we already test